src/ucai/core/envs/databricks_env_vars.py

```diff
@@ -0,0 +1,55 @@
+import os
+from typing import Optional
+
+
+class _EnvironmentVariable:
+    def __init__(self, name: str, default_value: str, description: Optional[str] = None):
```

Comment:
```suggestion
    def __init__(self, name: str, default_value: str, description: str):
```

I think description should not be optional.